core/arch/arm/plat-ti/conf.mk

```diff
@@ -7,27 +7,38 @@ CFG_SECURE_DATA_PATH ?= y
 
 ifeq ($(CFG_SECURE_DATA_PATH),y)
 CFG_TEE_SDP_MEM_SIZE ?= 0x00400000
-else
-CFG_TEE_SDP_MEM_SIZE ?= 0x0
```

Comment:
This change seems unrelated to the rest.

Does this break the CFG_SECURE_DATA_PATH == y case to have CFG_TEE_SDP_MEM_SIZE undefined like this?

Reply:
True. That change is not required. `CFG_TEE_SDP_MEM_SIZE` is not used even if defined when SDP is disable
Tell me if you prefer I restore to removed code or not.